tests/automation_framework/tests/worker_receipt/test_receipt.py

```diff
@@ -63,7 +61,6 @@ def test_work_order_create_receipt_success(self):
 
     @pytest.mark.work_order_retrieve_receipt
```

Comment:
need to remove work_order_retrieve_receipt marker

Reply:
done in the latest commit, had some other receipt API markers, removed those as well.